eslint.config

```diff
@@ -0,0 +1,10 @@
+{
+  "parserOptions": { "ecmaVersion": 2018 },
+  "extends": "airbnb-base",
+    rules: {
+    'no-shadow': 0,
+    'no-param-reassign': 0,
+    'eol-last': 0,
+  }
+}
+
```

Comment:
What is this file doing here? 🤔 You are supposed to use `.eslintrc.json` which you already have. Maybe you want to remove this one.